tailwind.config.js

```diff
@@ -1,7 +1,84 @@
 export default {
   content: ["./index.html", "./src/**/*.{js,jsx,ts,tsx}"],
+  darkMode: "class",
   theme: {
-    extend: {},
+    extend: {
+      perspective: {
+        1000: "1000px",
+      },
+      rotate: {
+        "y-180": "rotateY(180deg)",
+      },
+      colors: {
+        primary: {
```

Comment:
can we add a color name instead of primary? it will be easier to understand the code.